src/MicrosoftTeams.ts

```diff
@@ -588,6 +590,49 @@ namespace microsoftTeams {
     };
   }
 
+  /**
+   * enable print capability
+   */
+  export function enablePrintCapability(): void {
+    ensureInitialized();
+    printCapabilityEnabled = true;
+    /**
+     * adding ctrl+P and cmd+P handler
+     */
+    document.addEventListener("keydown", function(event: KeyboardEvent): void {
+      if ((event.ctrlKey || event.metaKey) && event.keyCode === 80) {
+        microsoftTeams.print();
+        event.cancelBubble = true;
+        event.preventDefault();
+        event.stopImmediatePropagation();
+      }
+    });
+  }
+
+  /**
+   * Registers a handler for print.
```

Comment:
Is this comment correct? It looks like this actually calls (not registers) the custom print handler or window.print.

Reply:
thanks. will update